### ISSUE TITLE
Missing `Blob` type for body in runtime fetchJson

### ISSUE THREAD

**Issue by tmueller** [cellular/oazapfts#54]
Hi oazapfts team,

when using an upload endpoint, that returns JSON, code is generated, that does not compile. 

The Problem occurs, if the enpoints requestBodys schema has format `binary` and its response type is `application/json`. Then a function is generated, where the body parameter is expected to be `Blob`. This body parameter is passed on to `oazapfts.fetchJson`, where body is typed as `string | FormData` without support for `Blob`. 

A [recent change](https://github.com/cellular/oazapfts/blob/81513f814302c76a8254865eaf2e0b0aa0bb9b2d/src/codegen/generate.ts#L401) introduced the change to Blob when the requestBodys schema has format binary.

I think, the Problem could be solved by widening the type of the body parameter in fetchJson. The underlying implementation should be able to cope.

Thanks for creating such a nice package and making it open source.

**Comment by tmueller** [comment]
With help of [the petstore example](https://github.com/cellular/oazapfts/blob/310117038b6a5119c8a121063d9f4d6f36f43b59/demo/petstore.json#L388) a similar scenario is tested. The behaviour I described does not occur during testing and all tests succeed. 

The reason is, that in the given example the requestBodies content is [multipart/form-data](https://github.com/cellular/oazapfts/blob/310117038b6a5119c8a121063d9f4d6f36f43b59/demo/petstore.json#L406). Oazapfts has a special handling for this case.

As of [swagger docs for file upload](https://swagger.io/docs/specification/describing-request-body/file-upload/), `multipart/form-data` might not be used. It is allowed to specify the requests media type, e.g. `image/png` for images of type png. This leads to the error I described above.

I'd be happy to submit a PR, I just need some more information:
Should I place an initially failing test in petstore.json?
Is widening the type of property `body` in [FetchRequestOpts](https://github.com/cellular/oazapfts/blob/25d67ba6519be8ad41661a15a412d5167b847c82/src/runtime/index.ts#L12) to include `Blob` considered the correct fix or are there any further undesired consequences?
